concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-235	235	Perlhuhn	Perlhuhn	guinea fowl	456	GUINEA FOWL
+Steinthal-1875-1565-236	236	Gans	Gans	goose	1187	GOOSE
+Steinthal-1875-1565-237	237	Ente	Ente	duck	1360	DUCK
+Steinthal-1875-1565-238	238	Küchlein	Küchlein	chick	2513	CHICKEN CHICK
+Steinthal-1875-1565-239	239	Henne hat ein Ei gelegt.	Henne hat ein Ei gelegt.	Hen laid an egg.		
+Steinthal-1875-1565-240	240	Taube	Taube	dove	1853	DOVE
+Steinthal-1875-1565-241	241	Schildkröte	Schildkröte	turtle	112	TURTLE
+Steinthal-1875-1565-242	242	Krokodil (Alligator)	Krokodil (Alligator)	crocodile	1857	CROCODILE
+Steinthal-1875-1565-243	243	Eidexe	Eidexe	lizard	632	LIZARD
+Steinthal-1875-1565-244	244	Frosch	Frosch	frog	503	FROG
+Steinthal-1875-1565-245	245	Kröte	Kröte	toad	894	TOAD
+Steinthal-1875-1565-246	246	Schlange, kriecht (deren Arten).	Schlange	snake	730	SNAKE
+Steinthal-1875-1565-247	247	Schlange, kriecht (deren Arten).	Schlange kriecht.	Snake crawls.		
+Steinthal-1875-1565-248	248	Fisch (Arten) schwimmt.	Fisch schwimmt.	Fish swims.		
+Steinthal-1875-1565-249	249	Krebs	Krebs	cancer, crab	2348	CRAB
+Steinthal-1875-1565-250	250	Skorpion	Skorpion	scorpion	1538	SCORPION
+Steinthal-1875-1565-251	251	Spinne	Spinne	spider	843	SPIDER
+Steinthal-1875-1565-252	252	Käfer	Käfer	bug	1027	BUG
+Steinthal-1875-1565-253	253	Ameise (Arten)	Ameise	ant	587	ANT
+Steinthal-1875-1565-254	254	Heuschrecke	Heuschrecke	locust	810	LOCUST
+Steinthal-1875-1565-255	255	Biene	Biene	bee	665	BEE
+Steinthal-1875-1565-256	256	Honig	Honig	honey	942	HONEY
+Steinthal-1875-1565-257	257	Wachs	Wachs	beewax	1931	BEESWAX
+Steinthal-1875-1565-258	258	Stachel	Stachel	stinger		
+Steinthal-1875-1565-259	259	Wespe	Wespe	wasp	1517	WASP
+Steinthal-1875-1565-260	260	Fliege sticht.	Fliege sticht.	Fly stings.		
+Steinthal-1875-1565-261	261	Mücke	Mücke	mosquito	1509	MOSQUITO
+Steinthal-1875-1565-262	262	Laus	Laus	louse	1392	LOUSE
+Steinthal-1875-1565-263	263	Floh	Floh	flea	232	FLEA
+Steinthal-1875-1565-264	264	Wanze	Wanze	bed bug	1949	BEDBUG
+Steinthal-1875-1565-265	265	Schmetterling	Schmetterling	butterfly	1791	BUTTERFLY
+Steinthal-1875-1565-266	266	Raupe	Raupe	caterpillar	2247	CATERPILLAR
+Steinthal-1875-1565-267	267	Wurm	Wurm	worm	1219	WORM
+Steinthal-1875-1565-268	268	Seide	Seide	silk	1641	SILK
+Steinthal-1875-1565-269	269	Schnecke	Schnecke	snail	1543	SNAIL
+Steinthal-1875-1565-270	270	Auster	Auster	oyster	2830	OYSTERS
+Steinthal-1875-1565-271	271	Männchen	Männchen	male (of animals)	2263	MALE (OF ANIMAL)
+Steinthal-1875-1565-272	272	Weibchen	Weibchen	female (of animals)	2262	FEMALE (OF ANIMAL)
+Steinthal-1875-1565-273	273	Fleisch	Fleisch	flesh, meat	2615	FLESH OR MEAT
+Steinthal-1875-1565-274	274	Fett	Fett	fat	323	FAT (ORGANIC SUBSTANCE)
+Steinthal-1875-1565-275	275	Milch	Milch	milk	635	MILK
+Steinthal-1875-1565-276	276	Butter	Butter	butter	1245	BUTTER
+Steinthal-1875-1565-277	277	Käse	Käse	cheese	1569	CHEESE
+Steinthal-1875-1565-278	278	Haar	Haar	hair	2678	HAIR OR FUR
```

Comment:
Can't be disambiguated?

Reply:
We could go for 580 FUR because the following concepts (not a very strong argument, but the only category indicator in this list) all refer to animal body parts. I was just wondering why he didn't go for "Fell" then.